apiv6/kentikapi/httputil/httputil_test.go

```diff
@@ -106,6 +107,47 @@ func TestRetryingClientWithSpyHTTPTransport_Do(t *testing.T) {
 	}
 }
 
+func TestNewRetryingClient_Do_HandlesRetryAfterHeader(t *testing.T) {
+	const retryAfterValue = 1 * time.Second
+	httpErrors := []int{
+		http.StatusTooManyRequests,
+		http.StatusServiceUnavailable,
+		http.StatusOK,
+	}
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		header := time.Now().Add(retryAfterValue).Format(time.RFC1123)
+		w.Header().Set("Retry-After", header)
+		http.Error(w, "", httpErrors[0])
+		httpErrors = httpErrors[1:]
+	}))
+	defer ts.Close()
+
+	c := httputil.NewRetryingClient(httputil.ClientConfig{
+		RetryCfg: httputil.RetryConfig{
+			MaxAttempts: intPtr(5),
+			MinDelay:    durationPtr(10 * time.Second),
+			MaxDelay:    durationPtr(10 * time.Second),
+		},
+	})
+
+	req, err := retryablehttp.NewRequest(http.MethodGet, ts.URL, nil)
+	require.NoError(t, err)
+
+	// act
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	defer cancel()
+
+	//nolint:bodyclose
+	resp, err := c.Do(req.WithContext(ctx))
+
+	// assert
+	t.Logf("Got response: %v, err: %v", resp, err)
+
+	assert.NoError(t, err)
+	assert.Equal(t, 200, resp.StatusCode)
```

Comment:
```suggestion
	assert.Equal(t, http.StatusOK, resp.StatusCode)
```
Please use the same form as in test input (missed that earlier).

Reply:
Done